assignments/objects.js

```diff
@@ -11,37 +11,98 @@
 
 // Example format of an intern object: 1,examples@you.edu,Example,F
 const example = {
-  "id": 0,
-  "name": "Example",
-  "email": "examples@you.edu",
-  "gender": "F"
+	"id": 0,
+	"name": "Example",
+	"email": "examples@you.edu",
+	"gender": "F"
 }
 
 // Write your intern objects here:
 
+const newInterns = [
+	{
+		"id": 1,
+		"name": "Mitzi",
+		"email": "mmelloy0@psu.edu",
+		"gender": "F",
+	},
+	{
+		"id": 2,
+		"name": "Kennan",
+		"email": "kdiben1@tinypic.com",
+		"gender": "M",
+		kenanSpeak: function () {
+			return "Hello, my name is Kennan!";
+		},
```

Comment:
Good approach with these, adding them all into an array instead of defining a specific variable for each intern.

You could also define the functions inline if you wanted:
```js
newInterns[1].speak = function() {
  return "Hello, my name is Kennan!"
}
```

You could also refer to kennan's name here by also using the `this` keyword. What `this` refers to inside the function is the `newInterns[1]` object itself.
```js
newInterns[1].speak = function() {
  return `Hello, my name is ${this.name}`;
}
```

Reply:
Oh!! So you can add them on the fly instead of keying them into the actual object. That is pretty cool, I've never seen that before.